Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -380,11 +453,136 @@ private class GeometryEditorModel: ObservableObject {
     ///   - tool: The tool to draw with.
     ///   - geometryType: The type of geometry to draw.
     func startEditing(with tool: GeometryEditorTool, geometryType: Geometry.Type) {
+        selectedGraphic = nil
         configureGeometryEditorTool(tool, scaleMode: scaleMode)
         geometryEditor.tool = tool
         geometryEditor.start(withType: geometryType)
         isStarted = true
     }
+    
+    /// Starts editing a given graphic with the geometry editor.
+    /// - Parameter graphic: The graphic to edit.
+    func startEditing(withGraphic graphic: Graphic) {
+        guard let geometry = graphic.geometry else { return }
+        
+        switch geometry {
+        case is Point, is Multipoint:
+            geometryEditor.tool = VertexTool()
+        default:
+            break
+        }
```

Comment:
Is this needed?

Reply:
Yes, it sets the geometry editor tool based on the geometry. This logic is specified in the sample design implementation details.

Reply:
I'll ask the design author Andy why it is needed. Without these lines, the geometry editor still works, so I prefer removing them. Will let you know tomorrow if he gets back.

Reply:
Removed the unnecessary lines.

Reply:
After consulting bistromath team, this logic needs to be added back.